doc/configtables/industry.csv

```diff
@@ -36,3 +36,4 @@ MWh_MeOH_per_tMeOH,LHV,float,"The energy amount per ton of methanol. From `DECHE
 hotmaps_locate_missing,--,"{true,false}",Locate industrial sites without valid locations based on city and countries.
 reference_year,year,YYYY,The year used as the baseline for industrial energy demand and production. Data extracted from `JRC-IDEES 2015 <https://data.jrc.ec.europa.eu/dataset/jrc-10110-10001>`_
 oil_refining_emissions,tCO2/MWh,float,"The emissions from oil fuel processing (e.g. oil in petrochemical refinieries). The default value of 0.013 tCO2/MWh is based on DE statistics for 2019; the EU value is very similar."
+temporal_electricity_industry_load,--,"{true,false}", "Switch to enable the use of temporally resolved electricity industry load profiles from `FfE (Forschungsstelle für Energiewirtschaft) 2021 <https://github.com/PyPSA/pypsa-eur/pull/1875>`_. Default is false."

```

Comment:
Maybe add a comment that this is data for Germany that is projected to the rest of Europe (i.e. per industry same profile as seen in Germany).